core/src/main/scala/org/apache/spark/api/python/PythonHadoopUtil.scala

```diff
@@ -122,7 +122,7 @@ private[python] class JavaToWritableConverter extends Converter[Any, Writable] {
     obj match {
       case i: java.lang.Integer => new IntWritable(i)
       case d: java.lang.Double => new DoubleWritable(d)
-      case l: java.lang.Long => new LongWritable(l)
+      case v: java.lang.Long => new LongWritable(v)
```

Comment:
Hm, tough one. I understand the logic behind avoiding `l` because it looks like `1`. Here, it feels OK to use `l` though. `v` suggests `void` here, another primitive type. Maybe `n`? not sure. I'd argue we could leave instances like this alone, though then we have to remove the style rule.